resources/assets/coffee/react/profile-page/info.coffee

```diff
@@ -35,64 +35,61 @@ class ProfilePage.Info extends React.Component
 
     return keys
 
+  datas = ['twitter', 'skype', 'lastfm', 'playstyles']
```

Comment:
why is it located here and not inside `render`? Or even directly used for the map instead through a variable `['a', ...].map`.